Source/GTMSessionFetcher.m

```diff
@@ -819,11 +819,12 @@ - (void)beginFetchMayDelay:(BOOL)mayDelay
 #endif  // GTM_DISABLE_FETCHER_TEST_BLOCK
 
 #if GTM_BACKGROUND_TASK_FETCHING
-  // Background tasks seem to interfere with out-of-process uploads and downloads.
-  if (!self.skipBackgroundTask && !self.useBackgroundSession) {
+  // Background tasks seem to interfere with out-of-process uploads and downloads. Background
```

Comment:
The background fetch support isn't out of process, it is just about using a UIApplication background task to allow it to finish and fire callbacks.